src/complaints/InitiatedComplaint.js

```diff
@@ -0,0 +1,68 @@
+import { View, Text, StyleSheet, Image } from 'react-native'
+import React from 'react'
+
+const InitiatedComplaint = ({ ComplaintData }) => {
+  return (
+    <View style={styles.container}>
+        <View style={styles.closedIndicator}/>
+        <View style={styles.textContainer}>
+        <Text style={styles.complaintName}>{ComplaintData.complaintName}</Text>
+        <Text style={styles.otherInfo}>{ComplaintData.otherInfo}</Text>
+        </View>
+
+        <View style={styles.imageContainer}>
+      <Image
+        source={ComplaintData.imageSource}
+        style={styles.image}
+        resizeMode="cover"
+      />
+      </View>
+
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+    container: {
+        flexDirection: 'row',
+        alignItems: 'center',
+        backgroundColor: '#cbd5e1',
+        borderRadius: 10,
+        // overflow: 'hidden',
+        width:400,
+        height:200,
+        marginBottom:10,
+      },
+      closedIndicator: {
+        width: 6,
+        backgroundColor: '#BDAB04',
+        height: '100%', // Makes the line stretch to the height of the container
+      },
+      textContainer: {
+        flex: 1,
+        padding: 10,
+        justifyContent: 'center',
+      },
+      complaintName: {
+        fontSize: 20,
+        color: '#333',
+      },
+      otherInfo: {
+        fontSize: 16,
+        color: '#333',
+      },
+      imageContainer: {
+        flex: 1,
+        justifyContent: 'center',
+        alignItems: 'center',
+        padding: 8,
+      },
+      image: {
+        width: '100%',
+        height: '100%',
+        resizeMode: 'contain',
+        borderRadius: 8,
+      },    
+});
+
+export default InitiatedComplaint
```

Comment:
** Hi there! I'd like to propose a suggestion for organizing styles within the complaints directory.**

** Could we consider centralizing all complaint-related styles into a single file, such as styles.js, and then import specific styles from that file as needed within individual components?**

Here are the potential benefits of this approach:

- Enhanced maintainability: Congregating styles in one place simplifies tracking and updating them.
- Improved organization: A centralized file fosters a clear structure and reduces style scattering.
- Potential performance gains: Reusing styles from a single file can minimize redundant CSS generation.
- Modularity and reusability: Isolating styles promote their easy transfer between components.
- Enhanced readability: A dedicated styles file facilitates code comprehension and visual consistency.

Implementation example:

1. Create styles.js: 
// styles.js
```
export const complaintHeading = `
  /* Styles for complaint headings */
`;

export const complaintName = `
  /* Styles for complaint body text */
`;
// ... other complaint-related styles
```
2. Import in components:
// ComplaintForm.js
```
import { complaintHeading, complaintName } from './styles.js';

function ComplaintForm() {
  return (
    <div className={complaintHeading}>Complaint Form</div>
    <Text style={styles.complaintName}>{ComplaintData.complaintName}</Text>
      {/* Complaint content */}
    </div>
  );
}
```

**I'm open to discussing this further if you have any questions or concerns! **

Reply:
Okay That`s great to hear soon that should be implemented.